src/Tween.js

```diff
@@ -542,3 +575,87 @@ TWEEN.Easing.Bounce.EaseInOut = function ( k ) {
 	return TWEEN.Easing.Bounce.EaseOut( k * 2 - 1 ) * 0.5 + 0.5;
 
 };
+
+
+TWEEN.Interpolation = { Utils: {
+
+	Linear: function ( p0, p1, t ) {
+
+		return ( p1 - p0 ) * t + p0;
+
+	},
+
+	Bernstein: function ( n , i ) {
+
+		var fc = TWEEN.Interpolation.Utils.Factorial;
+		return fc( n ) / fc( i ) / fc( n - i );
+
+	},
+
+	Factorial: ( function () {
+
+		var a = [ 1 ];
+
+		return function ( n ) {
+
+			var s = 1, i;
+			if ( a[ n ] ) return a[ n ];
+			for ( i = n; i > 1; i-- ) s *= i;
+			return a[ n ] = s;
+
+		}
+
+	} )(),
+
+	CatmullRom: function ( p0, p1, p2, p3, t ) {
```

Comment:
@mrdoob @egraether If using CatmullRom, then we need to supply no more than 4 numbers in an array value? Is there not a more dynamic way?